### ISSUE TITLE
fsl: topup -> applytopup (passing prefixes)

### ISSUE THREAD

**Issue by kaitj** [childmindresearch/niwrap#119]
Might have just missed this one, but from what I am able to gather:

* topup will output `fieldcoef` and `movpar` files that can be accessed.
* these are files that are typically accessed in a subsequent step `applytopup`, via the same prefix that is used

Issue is stemming from `applytopup` wanting that prefix (inclusive of any parent directories since I think it reads the path directly). This doesn't quite fall into outputs, but it is something relevant that is needed.  For now, I can just use one of the relevant outputs from topup and strip the name from the last "_":

```python
topup = fsl.topup(
    ...
)

fsl.applytopup(
    ...,
    topup="".join(part for part in str(topup.movpar).split("_")[:-1]
)
```

And unrelated to this issue, I wonder if there are similar calls in other fsl commands or ANTs.  

~Actually more I think about it, this really isn't a niwrap / styx issue since that is doing its job of wrappingthe tool and more of a "workaround with python".  We should probably note this in the eventual documentation - feel free to close!~

**Comment by kaitj** [comment]
Updating before I forget now that i am testing a workflow that runs this. I think it’s fine for local runners but this is also an issue for the containers that bind the io files. Passing the prefix as string is fine, but the files aren’t found because they aren’t bound.

**Comment by kaitj** [comment]
This issue was fixed as a part of #146.
